contracts/HypERC721.sol

```diff
@@ -14,24 +14,21 @@ contract HypERC721 is ERC721EnumerableUpgradeable, TokenRouter {
      * @notice Initializes the Hyperlane router, ERC721 metadata, and mints initial supply to deployer.
      * @param _mailbox The address of the mailbox contract.
      * @param _interchainGasPaymaster The address of the interchain gas paymaster contract.
-     * @param _interchainSecurityModule The address of the interchain security module contract.
      * @param _mintAmount The amount of NFTs to mint to `msg.sender`.
      * @param _name The name of the token.
      * @param _symbol The symbol of the token.
      */
     function initialize(
         address _mailbox,
         address _interchainGasPaymaster,
-        address _interchainSecurityModule,
         uint256 _mintAmount,
         string memory _name,
         string memory _symbol
     ) external initializer {
         // transfers ownership to `msg.sender`
         __HyperlaneConnectionClient_initialize(
             _mailbox,
-            _interchainGasPaymaster,
-            _interchainSecurityModule
```

Comment:
Are we cutting the ISM concept for v0?

Reply:
good question

Reply:
I think he did this for the deploy but not sure we actually want to publish it this way

Reply:
No, but currently the router asserts that you can't initialize this to an address that is not a contract which means 0x0 is not valid. IMO we should have multiple initializers and not require the passing of ISMs if the default is to be used

Reply:
Users can always set via the explicit setISM function